### PR TITLE
Prevent message from being sent without subject and/or body from dev portal

### PR DESCRIPTION
**Pull request by nidhi-soni1104**
**Which issue(s) this PR fixes**

https://issues.redhat.com/browse/THREESCALE-8393

### CODE REVIEW

**Review by jkeam — APPROVED** [review]
looks good

**Comment by jkeam** [comment]
I'm not sure if you would normally include tests for this though.  For example:

https://github.com/3scale/porta/pull/2983/files

File: features/old/applications/providers/bulk_operations/send_emails.feature [hunk 1]
```diff
@@ -76,3 +76,13 @@ Feature: Mass email bulk operations
     Given the email will fail when sent
     And I press "Send" and I confirm dialog box within colorbox
     Then I should see the bulk action failed with account "jane"
+
+Scenario: Emails can't be sent without subject
```
Comment (jkeam) [inline features/old/applications/providers/bulk_operations/send_emails.feature:80]: Awesome, glad you were able to add a test for this.

Comment (nidhi-soni1104) [reply features/old/applications/providers/bulk_operations/send_emails.feature:80]: I added test but it keeps failing,  I added its path in the paths.rb file.
when I run the test it doesn't show any specific error but just failed.
I have added similar kind of test before which worked but not for this case.
Not sure if it works for files html.liquid also . If you have any idea about this feel free to share so I can check.

**Comment by nidhi-soni1104** [comment]
@josemigallas can you review this PR?

**Comment by josemigallas** [comment]
> @josemigallas can you review this PR?

@nidhi-soni1104 I can't really review this because I'm confused what the PR is about. The title and the linked JIRA are inconsistent and so is the description. The updated form belongs to Developer portal messages, however the new test is about bulk_operations.

Could you please tidy it up so it's clearer what to review?

**Comment by nidhi-soni1104** [comment]
@josemigallas why you put it in draft

**Comment by josemigallas** [comment]
> @josemigallas why you put it in draft

When a PR is still WIP it is better to make it a draft to avoid accidental merge and unwanted reviews. This one is still not ready for review.

**Pull request closed without merging**